packages/ng/schematics/lu-button/tests/output/simple-cases.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { IconComponent } from '@lucca-front/ng/icon';
+import { ButtonComponent } from '@lucca-front/ng/button';
+
+@Component({
+	selector: 'lu-test',
+	standalone: true,
+	template: `
+		<button
+		 luButton="outlined" block class="u-textCenter u-textNeutral"
```

Comment:
```suggestion
			luButton="outlined" block class="u-textCenter u-textNeutral"
```

The indent detection does not work anymore?